### PR TITLE
Low latency unit tests

### PR DESCRIPTION
**Pull request by CaitlinOCallaghan**
Unit tests for LL-DASH support features

## Test Cases

`./chunking_unittest --gtest_filter="ChunkingHandlerTest.LowLatencyDash"`

`./media_event_unittest --gtest_filter="MpdNotifyMuxerListenerTest.LowLatencyDash"`

`./mpd_unittest --gtest_filter="PeriodTest.LowLatencyDashMpdGetXml"`
`./mpd_unittest --gtest_filter="SimpleMpdNotifierTest.NotifyAvailabilityTimeOffset"`
`./mpd_unittest --gtest_filter="SimpleMpdNotifierTest.NotifySegmentDuration"`
`./mpd_unittest --gtest_filter="LowLatencySegmentTest.LowLatencySegmentTemplate"`

Note, packager_test must be run from the main project directory
`./out/Release/packager_test --gtest_filter="PackagerTest.LowLatencyDashEnabledAndUtcTimingNotSet"`
`./out/Release/packager_test --gtest_filter="PackagerTest.LowLatencyDashEnabledAndUtcTimingNotSet"`

### CODE REVIEW

File: packager/app/packager_main.cc [hunk 1]
```diff
@@ -329,18 +329,6 @@ base::Optional<PackagingParams> GetPackagingParams() {
   chunking_params.segment_sap_aligned = FLAGS_segment_sap_aligned;
   chunking_params.subsegment_sap_aligned = FLAGS_fragment_sap_aligned;
 
-  if (chunking_params.is_low_latency_dash &&
```
Comment (ngarg17) [inline packager/app/packager_main.cc]: Isn't this change already covered in your other pull request?

Comment (CaitlinOCallaghan) [reply packager/app/packager_main.cc]: It is - I made this unit test branch off of the `restirct-low-latency-inputs` branch seen in the other PR. This difference will disappear once the `restirct-low-latency-inputs` PR is merged into the `low-latency-dash-support-staging branch`

**Comment by CaitlinOCallaghan** [comment]
Converting to a draft while I work out the Build and Test PR / Build and test win Debug shared (pull_request) check.

**Comment by kevleyski** [comment]
Do those checks fail without the changes too?

**Comment by CaitlinOCallaghan** [comment]
> Do those checks fail without the changes too?

There is a rounding issue with the availabilityTimeOffset float on the Windows build. Everything that I tested manually was done on an x86 Ubuntu VM.

**Comment by CaitlinOCallaghan** [comment]
All checks are now passing! :)

**Review by ngarg17 — APPROVED** [review]
Looks good to me.

**Review by joeyparrish — APPROVED** [review]
Very thorough!  Thank you!

**Review by stephenlindauer — APPROVED** [review]
Looks good!

**Pull request merged**